code/rsapss/Hacl.RSAPSS.fst

```diff
@@ -84,3 +84,9 @@ val rsapss_pkey_verify:
 let rsapss_pkey_verify a modBits eBits nb eb saltLen sgntLen sgnt msgLen msg =
   RI.rsapss_pkey_verify (ke modBits) a modBits
     (load_pkey modBits) (rsapss_verify a modBits) eBits nb eb saltLen sgntLen sgnt msgLen msg
+
+[@ (Comment "Miscellaneous functions")]
+
```

Comment:
you need to remove a new line here. otherwise, you'll get `Syntax error`:
```
[@ (Comment "Miscellaneous functions")]
val mgf_hash: a:Hash.algorithm{S.hash_is_supported a} -> Hacl.Impl.RSAPSS.MGF.mgf_hash_st a
```

Reply:
You can also expose this function by adding `Hacl.Impl.RSAPSS.MGF` to your `RSAPSS_BUNDLE`